src/main/java/org/radarcns/biovotion/BiovotionDeviceManager.java

```diff
@@ -128,24 +162,35 @@ public BiovotionDeviceManager(Context context, DeviceStatusListener biovotionSer
         this.energyTable = dataHandler.getCache(topics.getEnergyTopic());
         this.temperatureTable = dataHandler.getCache(topics.getTemperatureTopic());
         this.gsrTable = dataHandler.getCache(topics.getGsrTopic());
+        this.accelerationTable = dataHandler.getCache(topics.getAccelerationTopic());
+        this.ledCurrentTable = dataHandler.getCache(topics.getLedCurrentTopic());
         this.batteryTopic = topics.getBatteryStateTopic();
 
         this.biovotionService = biovotionService;
         this.context = context;
 
         this.bleServiceConnectionIsBound = false;
 
+        this.gapExecutor = new ScheduledThreadPoolExecutor(1);
+        this.utcExecutor = new ScheduledThreadPoolExecutor(1);
```

Comment:
Perhaps use `Executors.newSingleThreadScheduledExecutor();` instead, for consistency with other packages?